src/components/Messages/Messages.jsx

```diff
@@ -37,5 +37,5 @@ function Messages() {
 // Invoke repetedly obsolete messages deletion (should be in slice file but I cannot find a way to access the store from there)
 window.setInterval(() => {
   store.dispatch(handle_obsolete());
-}, 1000);
+}, 60000);
```

Comment:
handle_obsolete was running every 0.1 seconds flooding redux with state updates. made it hard to debug. Once every 60 seconds cleans up redux state for devs && would seem to have no negative impacts for a production user.